lib/src/lightclient.rs

```diff
@@ -863,11 +863,68 @@ impl LightClient {
                 })
                 .for_each( |note| {
                     if note["spent"].is_null() && note["unconfirmed_spent"].is_null() {
-                        unspent_notes.push(note);
+                        unspent_sapling_notes.push(note);
                     } else if !note["spent"].is_null() {
-                        spent_notes.push(note);
+                        spent_sapling_notes.push(note);
                     } else {
-                        pending_notes.push(note);
+                        pending_sapling_notes.push(note);
+                    }
+                });
+        }
+
+        let mut unspent_orchard_notes: Vec<JsonValue> = vec![];
+        let mut spent_orchard_notes: Vec<JsonValue> = vec![];
+        let mut pending_orchard_notes: Vec<JsonValue> = vec![];
+
+        {
+            // Same thing but with orchard. TODO: Dry
+            let spendable_address: HashSet<String> = self
+                .wallet
+                .keys()
+                .read()
+                .await
+                .okeys()
+                .into_iter()
+                .filter(|k| OrchardSpendingKey::try_from(&k.key).is_ok())
+                .map(|k| k.unified_address.encode(&self.config.chain))
+                .collect();
+
+            self.wallet.transaction_context.transaction_metadata_set.read().await.current.iter()
+                .flat_map( |(transaction_id, wtx)| {
+                    let spendable_address = spendable_address.clone();
+                    wtx.orchard_notes.iter().filter_map(move |nd|
+                        if !all_notes && nd.spent.is_some() {
```

Comment:
`all_notes` is a bool that should be named `include_unspect_notes`.  It's a bool.